src/panels/lovelace/hui-root.ts

```diff
@@ -120,6 +120,12 @@ class HUIRoot extends LitElement {
                   <div main-title>
                     ${this.config.title ||
                     this.hass!.localize("ui.panel.lovelace.editor.header")}
+                    ${this.config.append_view_title ?
+                      html`
+                        -
+                        ${this.config.views[this._curView ?? 0].title}
```

Comment:
if we dont have a current view we should just not append

Reply:
When the view selected is the first one in the list. _curView is undefined. Don't know if a bug or a feature. I force in that case to show the first one title.

A view must be always be selected, not?